src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,12 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &__item {
+    width: 16px;
+    height: 16px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+  }
```

Comment:
You have missed setting the background-position property for the star image. According to the checklist, it should be set to 'center'. This will ensure the image is always centered within its container.